### PR TITLE
Announcing 2 new detections

### PR DESCRIPTION
**Pull request by unknownhad**
Announcing 2 new detections

### CODE REVIEW

File: content/waf/change-log/scheduled-changes.md [hunk 1]
```diff
@@ -24,14 +24,24 @@ layout: list
   </thead>
   <tbody>
     <tr>
+      <td>2022-05-05</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]: Must we wait for May 5 before announcing these changes?

Comment (unknownhad) [reply content/waf/change-log/scheduled-changes.md]: My bad it should be today's date.
Updating it.

Comment (unknownhad) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>2022-05-03</td>
```

File: content/waf/change-log/scheduled-changes.md [hunk 2]
```diff
@@ -24,14 +24,24 @@ layout: list
...
+      <td>2022-05-09</td>
+      <td>Cloudflare Specials</td>
       <td>N/A</td>
+      <td>100528</td>
+      <td>Code Injection - CVE:CVE-2022-29078</td>
       <td>N/A</td>
+      <td>Block</td>
+    </tr>
+        <tr>
+      <td>2022-05-05</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>2022-05-03</td>
```
The same here, I guess.

**Review by pedrosousa — APPROVED** [review]

File: content/waf/change-log/scheduled-changes.md [hunk 3]
```diff
@@ -24,14 +24,24 @@ layout: list
   </thead>
   <tbody>
     <tr>
+      <td>2022-05-03</td>
+      <td>2022-05-09</td>
+      <td>Cloudflare Specials</td>
       <td>N/A</td>
```
Comment (unknownhad) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>...02a9ee96</td>
```

File: content/waf/change-log/scheduled-changes.md [hunk 4]
```diff
@@ -24,14 +24,24 @@ layout: list
...
+      <td>100528</td>
+      <td>Code Injection - CVE:CVE-2022-29078</td>
       <td>N/A</td>
+      <td>Block</td>
+    </tr>
+        <tr>
+      <td>2022-05-03</td>
+      <td>2022-05-09</td>
+      <td>Cloudflare Specials</td>
       <td>N/A</td>
```
Comment (unknownhad) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>...422313d0</td>
```

**Pull request merged**